drivers/wifi/nrf700x/Kconfig

```diff
@@ -374,3 +374,26 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config NRF_WIFI_2G_BAND
```

Comment:
Current code only chooses one of these, so it could have been a choice. But I  think we should support multiple selections.

Reply:
Changed the NRF_WIFI_OP_BAND option in bit. It will support multiple selections.